Services/CO.CDP.Forms.WebApi/UseCase/UpdateFormSectionAnswersUseCase.cs

```diff
@@ -30,12 +30,30 @@ public async Task<bool> Execute((Guid formId, Guid sectionId, Guid answerSetId,
         }
         else if (sharedConsent.SubmissionState == Persistence.SubmissionState.Submitted)
         {
-            formRepository.ClearTracker();
```

Comment:
As commented on repository, this one is a no-go. Detaching entities either since we manually create new SharedConsents with children.

All entities before saving in this decision branch (with SharedConsent.SubmissionState == Submitted) should be of state Unchanged. That's because previously existing AnswerSets and their Answers for that SharedConsent are used to create a manual copy of it, so we don't want to modify it. And the newly, manually created copy is in Unchanged state too.

There is a new method fired just before saving to DB called CheckEntitiesStates that helps monitor the entity states during debugging. Please make sure to remove its usage, repository method and its interface declaration before merging this into Main.